src/main/java/seedu/address/logic/commands/AddEventCommand.java

```diff
@@ -34,42 +37,48 @@ public class AddEventCommand extends AddCommand {
             + PREFIX_EVENT_TIME + "TIME "
             + PREFIX_EVENT_VENUE + "VENUE "
             + PREFIX_EVENT_CELEBRITY + "CELEBRITY "
+            + PREFIX_EVENT_CONTACTS + "CONTACTS...\n"
             + "Example: " + COMMAND_WORD + " " + COMMAND_FIELD + " "
             + PREFIX_EVENT_NAME + "Oscars "
             + PREFIX_EVENT_TIME + "Sep 22 2024 1800 to 2200 "
             + PREFIX_EVENT_VENUE + "Hollywood "
-            + PREFIX_EVENT_CELEBRITY + "Sydney Sweeney ";
+            + PREFIX_EVENT_CELEBRITY + "Sydney Sweeney "
+            + PREFIX_EVENT_CONTACTS + "Alex Yeoh, Bernice Yu";
 
     public static final String MESSAGE_SUCCESS = "New Event added: %1$s";
     public static final String MESSAGE_DUPLICATE_EVENT = "This event  already exists in the address book";
     private final EventName eventName;
     private final Time time;
     private final Venue venue;
     private final String personName;
```

Comment:
maybe we can change this to celebrityName to be clearer now that we are adding contacts

Reply:
I see what you mean I will make the edits needed.

Reply:
astute observation, thanks fixed it already